contracts/nft/contracts/solana/tests/universal-nft-cycle.ts

```
@@ -0,0 +1,1398 @@
+                console.log(`      ⚠️  Invalid address was accepted (may be valid for some chains)`);
+            } catch (error) {
+                console.log(`      ✅ Correctly rejected invalid address format`);
+                console.log(`      📝 Error: ${error.message.split('\n')[0]}`);
+            }
+
+            // Test origin system with invalid chain ID
+            const invalidOriginTokenId = 888888;
+            const invalidChainId = 999999; // Unsupported chain
+            const [invalidOriginPda] = PublicKey.findProgramAddressSync(
+                [
+                    Buffer.from("nft_origin"),
```

Biome (2.1.2)

[error] Shouldn't redeclare 'invalidChainId'. Consider to delete it or rename it.

'invalidChainId' is defined here:


(lint/suspicious/noRedeclare)